install.js

```diff
@@ -22,12 +24,28 @@ var paths = {
   win32: path.join(__dirname, './dist/electron.exe')
 }
 
+var cache = {
+  darwin: path.join(getHomePath, './.electron'),
+  linux: path.join(getHomePath, './.electron'),
+  win32: path.join(getHomePath, './.electron')
+}
```

Comment:
Oh, I suppose we don't. It's a relic from my first attempt at saving the entire file.